packages/opentelemetry-instrumentation-openai/tests/traces/test_functions.py

```diff
@@ -60,11 +63,17 @@ def test_open_ai_function_calls(exporter, openai_client):
         == "What's the weather like in Boston?"
     )
     assert (
-        open_ai_span.attributes[f"{SpanAttributes.LLM_REQUEST_FUNCTIONS}.0.name"]
+        open_ai_span.attributes[f"{SpanAttributes.LLM_REQUEST_TOOLS}.0.type"]
+        == "function"
+    )
+    assert (
+        open_ai_span.attributes[f"{SpanAttributes.LLM_REQUEST_TOOLS}.0.function.name"]
         == "get_current_weather"
     )
     assert (
-        open_ai_span.attributes[f"{SpanAttributes.LLM_REQUEST_FUNCTIONS}.0.description"]
+        open_ai_span.attributes[
```

Comment:
These tests are missing checks for the params; I can add them if necessary

Reply:
Yes! Can you please add this? @achang97 
I've released the updated semconv so tests should pass